packages/legend-application-studio/src/stores/editor-state/element-editor-state/connection/ConnectionEditorState.ts

```diff
@@ -102,6 +109,7 @@ export class RelationalDatabaseConnectionValueState extends ConnectionValueState
   override connection: RelationalDatabaseConnection;
   selectedTab = RELATIONAL_DATABASE_TAB_TYPE.GENERAL;
   databaseBuilderState: DatabaseBuilderState;
+  selectedPostProcessor: PostProcessorEditorState | undefined;
```

Comment:
hm, could you do me a favor and rename these to `postProcessorState` and `setPostProcessorState`? thanks! :pray: